Marlin/src/feature/tmc_util.cpp

```diff
@@ -588,6 +596,13 @@
             SERIAL_ECHOPGM("/256");
           }
           break;
+        case TMC_INTERPOLATE:
+          {
+            CHOPCONF_t r{0};
+            r.sr = st.CHOPCONF();
+            serialprint_truefalse(r.intpol);
```

Comment:
Why not just call `st.intpol()`?

Reply:
Because I failed to follow the inheritance chain to see that it was available for all drivers. I thought there were some that didn’t have `intpol()` due to a naive reading of the code.

Reply:
I’ll get a fix in tomorrow.